projects/demo/src/modules/components/button/button.module.ts

```diff
@@ -40,8 +41,9 @@ import {TuiButtonExample6} from './examples/6';
         TuiNotificationModule,
         InheritedDocumentationModule,
         TuiAddonDocModule,
-        RouterModule.forChild(tuiGenerateRoutes(ExampleTuiButtonComponent)),
+        TuiLoaderModule,
```

Comment:
You've imported this module but there are no examples with our loader.

Make one example with `<tui-loader />`, please

Reply:
fixed